tests/features/user/login.feature

```diff
@@ -4,6 +4,8 @@ Feature: User login
   As a user of the website
   I need to be able to login with my credentials.
 
+  # Todo: needs to be adapted to use EU Login in ISAICP-5760.
+  @wip
```

Comment:
Why adding these TODOs? We know that, in ISAICP-5760, we'll have to adapt them. Otherwise, in ISAICP-5760, the tests will fail.

Reply:
I added them because it took a lot of time to go over all these scenarios and identify which ones needed to be updated, after multiple full test runs. It will make it much easier to do the first pass of ISAICP-5760, since most things will already be fixed before running the first test. Also it will help to not miss any of these tests which are marked with `@wip`, since they won't run and won't fail.